network/p2p/p2p.go

```diff
@@ -250,13 +251,13 @@ func (n *p2pNetwork) listen(sub *pubsub.Subscription) {
 	}(sub)
 }
 
-// GetTopic return topic by validator public key
-func (n *p2pNetwork) GetTopic(msg *proto.SignedMessage) (*pubsub.Topic, error) {
-	if msg.Message == nil || msg.Message.ValidatorPk == nil{
+// getTopic return topic by validator public key
+func (n *p2pNetwork) getTopic(validatorPK []byte) (*pubsub.Topic, error) {
```

Comment:
Should be public, you sure its not failing?

Reply:
why should it be public? it's not part of the network interface.